AlphaWallet/Foundation/Formatter.swift

```diff
@@ -66,6 +66,7 @@ struct Formatter {
 
     static let `default`: NumberFormatter = {
         let formatter = NumberFormatter()
+        formatter.locale = Locale(identifier: "en_US")
```

Comment:
Maybe we shouldn't set it here and use one of the other formatters instead? `.shortCrypto` maybe?

Reply:
i wasn't sure in applying Locale for `default` NumberFormatter, updated it

Reply:
I mean, can we not add this line at all?

> formatter.locale = Locale(identifier: "en_US")

since `basicNumberFormatter()` already add set it.